achilles-core/src/main/java/info/archinnov/achilles/internal/helper/LoggerHelper.java

```diff
@@ -31,4 +38,22 @@ public String apply(Field field) {
 			return field.getName();
 		}
 	};
+
+    public static List<Object> replaceByteBuffersByHexString(Object... values) {
+        ArrayList<Object> boundValues = new ArrayList<>(Arrays.asList(values));
+
+        for (int valuePos = 0; valuePos < boundValues.size(); valuePos++) {
+            Object boundValue = boundValues.get(valuePos);
+            if (boundValue instanceof ByteBuffer) {
+                ByteBuffer byteBuffer = (ByteBuffer) boundValue;
+                byte[] extracted = new byte[min(byteBuffer.remaining(), HEX_STRING_LOG_LIMIT)];
+                byteBuffer.get(extracted).rewind();
+                boundValues.set(valuePos, Bytes.toHexString(extracted) + ((byteBuffer.remaining() > HEX_STRING_LOG_LIMIT) ? "..." : ""));
+            } else if (boundValue instanceof byte[]) {
+                byte[] byteArray = (byte[]) boundValue;
+                boundValues.set(valuePos, Bytes.toHexString(((byteArray.length > HEX_STRING_LOG_LIMIT) ? Arrays.copyOfRange(byteArray, 0, HEX_STRING_LOG_LIMIT) : byteArray))  + ((byteArray.length > HEX_STRING_LOG_LIMIT) ? "..." : ""));
```

Comment:
can't you put the ```? "..." : ""``` logic that appearrs in both branches and even worse in the else as a nested ternary op in a method ?